README.md

```diff
@@ -111,7 +111,7 @@ Permissions that applications may use (names are subject to change):
 | Location | Use GPS and positioning. |
 | MediaIndexing | Access to Tracker to list files on device. If you have access to a data directory, you may want to use also this. |
 | Messages | Access to message data and to send SMS messages. |
-| Microphone | Record audio with microphone. Use Audio permission for playback of the recorded audio. |
+| Microphone | Record audio with microphone. Use Audio permission for playback of the recorded audio (but since Pulseaudio streams cannot be separated this enables also audio playback, which is subject to change ). |
```

Comment:
```suggestion
| Microphone | Record audio with microphone. Use Audio permission for playback of the recorded audio (but since Pulseaudio streams cannot be separated this enables also audio playback, which is subject to change). |
```

Reply:
Removed the extra space with c1cc3d824a657a2ad0ed329bd3cb0d987cd5c1e0 (small change so force pushed).